akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/ActorContext.scala

```diff
@@ -211,7 +211,7 @@ trait ActorContext[T] extends akka.actor.typed.ActorContext[T] {
 
   /**
    * This Actor’s execution context. It can be used to run asynchronous tasks
-   * like [[scala.concurrent.Future]] combinators.
+   * like [[scala.concurrent.Future]] operators.
```

Comment:
This is not about streams though, so this can be left as is

Reply:
Yeah, that sounds good to me. Sorry we didn't notice that when reviewing that PR

Reply:
the word `combinator` is used in Future docs once, and the word `operator` zero times, so technically sticking closer to its docs would be keeping combinator. It may be easier for our users though, esp in javadsl etc, to stick to the same name for all those -- to not have them think about it too much... I'd be fine with the change here